public/js/login.js

```diff
@@ -58,6 +56,40 @@ let loginUser = (event) => {
   })
 }
 
+//check to see if values in object are empty.
+let validateUser = (user) => {
+  if(user.email && user.user_password) {
+    return true;
+  }
+  return false;
+};
+
+let automaticSignIn = () => {
+  let checkUser;
+  if(localStorage.getItem("token")) {
+    checkUser = localStorage.getItem("token");
+    fetch(`/signedin`, {
+      method:"GET",
+      headers: {
+        "Content-Type": "application/json",
+        "token": checkUser
+      }
+    })
+    .then(response => 
+        response.text())
+    .then(message => {
+      if(message === 'is user') {
+          loginForm.classList.remove("hidden");
+      }
+      else {
+        loginForm.reset();
```

Comment:
This logic does not make sense  The login form should be blank already, and getLevelData is called later.  Remove the else statement.  If the message === "is user", then showInstructions().

Reply:
Are you saying to do else {
 showLoginForm()
getLevelData()
}

Reply:
if the message === "is user", then showInstructions().
no else